frontend/src/components/app/LoginForm.js

```diff
@@ -152,6 +157,9 @@ class LoginForm extends Component {
 
     request
       .then((response) => {
+        response.data.status === 502
+          ? this.props.dispatch(badGateway(BAD_GATEWAY_ERROR))
```

Comment:
instead of having `badGateway` action why not having `connectionError` action which takes `errorType` as parameter?